src/bci_build/package/appcontainers.py

```diff
@@ -486,3 +486,92 @@ def _get_nginx_kwargs(os_version: OsVersion):
     )
     for os_version in (OsVersion.TUMBLEWEED,)
 ]
+
+
+_BASE_PODMAN_OSC_CMD = (
+    "podman run --rm -it --privileged "
```

Comment:
Does it need to be privileged? Or could it get away with only certain capabilities to limit further damages?

Reply:
Yes, I'm afraid it is necessary for builds to succeed. I haven't so far found a way how to run a `osc build` without the `--privileged` flag.